docs/en/extending-doris/logstash.md

```diff
@@ -0,0 +1,198 @@
+---
```

Comment:
You need to add this doc title in the side bar:
open the `docs/.vuepress/sidebar/en.js` and search `extending-doris/`